main.py

```diff
@@ -770,9 +887,72 @@ def clean_up_db(self):
             if len(verification[email]['verified']) == 1 and len(verification[email]['unverified']) >= 1:
                 db.delete(verification[email]['unverified'])
 
+class CronHandler(BaseHandler):
+    def get(self, job): # On url invoke
+        if job == "lunch":
+            update_lunch.read_lunches()
+            self.response.write("Success")
+
+class PrivacyHandler(BaseHandler): # Change and view privacy settings
+    def load_obj(self):
+        id = self.check_id()
+        if id is None:
+            return None
+
+        logging.info(id)
+
+        email = convert_id_to_email(id)
+
+        logging.info(email)
+
+        user_obj_query = db.GqlQuery("SELECT * FROM User WHERE email = :1", email)
+        for user_obj in user_obj_query:
```

Comment:
This is kind of a weird construct. I think you can do user_obj_query.next() instead.